include/nn/ro.h

```diff
@@ -6,18 +6,66 @@
 #pragma once
 
 #include <nn/types.h>
+#include <nn/util/BinaryTypes.h>
+#include <elf.h>
 
 namespace nn {
 namespace ro {
 
-namespace rtld {
-struct ModuleObject;  // TODO find this object and implement it. Original `#include` name:
-                      // ModuleObject.hpp, full path name: nn::ro::rtld::ModuleObject
+namespace detail {
+    class RoModule {
+    public:
+        RoModule* next;
+        RoModule* prev;
+        union {
+            Elf64_Rel* rel_plt;
+            Elf64_Rela* rela_plt;
+            void* rel_rela_plt;
+        };
+        union {
+            Elf64_Rel* rel;
+            Elf64_Rela* rela;
+            void* rel_rela;
+        };
+        u8* moduleBaseAddr;
+        Elf64_Dyn* dynamic;
+        bool is_rela;
+        u8 pad_x31[7];
+        Elf64_Xword plt_size;
+        void (*initFunc)(void);
```

Comment:
```suggestion
        void (*initFunc)();
```
No need for void as this is C++.